end_to_end_tests/golden-master/my_test_api_client/api/tests.py

```diff
@@ -50,6 +50,66 @@ def get_user_list(
         raise ApiResponseError(response=response)
 
 
+def get_basic_list_of_strings(*, client: Client,) -> List[str]:
+
+    """ Get a list of strings  """
+    url = "{}/tests/basic_lists/strings".format(client.base_url)
+
+    headers: Dict[str, Any] = client.get_headers()
+
+    response = httpx.get(url=url, headers=headers,)
+
+    if response.status_code == 200:
+        return [str(item) for item in cast(List[str], response.json())]
```

Comment:
We probably don't need the `cast` here since we're converting to the correct type with `str`. I suppose it doesn't hurt though, so we can leave it in.